src/pages/pricing.tsx

```diff
@@ -350,7 +354,7 @@ const PricingPage: FunctionComponent = () => {
                                     price={<PriceItem amount={19} hasLimit={true} />}
                                     buttons={
                                         <ContactUsButton
-                                            href="https://sourcegraph.com/contact/request-info?form_submission_source=pricing-cody-enterprise"
+                                            href="/contact/pricing?form_submission_source=pricing-cody-enterprise"
```

Comment:
> Looking great! We updated the form so we need the Contact Sales on /pricing to include the form submission source again:
> 
> https://sourcegraph.com/contact/request-info?form_submission_source=pricing-cody-enterprise https://sourcegraph.com/contact/request-info?form_submission_source=pricing-enterprise https://sourcegraph.com/contact/request-info?form_submission_source=pricing-code-intelligence
> 
> Thank you!

On the current PR implementation we do have the form submission source

Reply:
@gitstart-sourcegraph Your implementation looks correct to me. I don't believe any changes are needed